devices/xiaomi.js

```diff
@@ -2164,4 +2252,20 @@ module.exports = [
             await endpoint1.read('aqaraOpple', [0x0125], {manufacturerCode: 0x115f});
         },
     },
+    {
+        zigbeeModel: ['lumi.airrtc.agl001'],
+        model: 'SRTS-A01',
+        vendor: 'Xiaomi',
+        description: 'Aqara Smart Radiator Thermostat E1',
+        fromZigbee: [fzLocal.aqara_trv, fz.thermostat],
+        toZigbee: [tzLocal.aqara_trv, tz.thermostat_occupied_heating_setpoint],
+        exposes: [e.switch().setAccess('state', ea.STATE_SET),
+            exposes.climate().withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
+                .withLocalTemperature(ea.STATE).withPreset(['manual', 'away', 'auto'], ea.STATE_SET),
+            e.child_lock(), e.window_detection(), e.valve_detection(),
+            e.away_preset_temperature(),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+        },
```

Comment:
Was this left empty on purpose? If not please remove.

Reply:
can be removed for now :)

Reply:
done